css/chat.css

```diff
@@ -341,7 +349,7 @@ html.no-supports-container.native body,
 }
 
 .chat-card-holder.unread:after {
-  content: '';
+  content: "";
```

Comment:
why this change?

Reply:
This is default behavior of eslint when we save the css file, it converts the **''** to **""** quotes

Reply:
I meant how it was not fixed earlier by eslint, @tonytlwu this seems to tech debt to me. We had created one ticket to address all eslint in past but we never tool that on priority to fix.